src/server/routes/enrichment/visualization/pathway-table.js

```diff
@@ -25,5 +29,11 @@ gmtPathwayData.split('\n').forEach( pathwayInfoLine => {
 
 pathwayInfoTable.delete('');
 
+/**
+ * handleFileUpdate
+ * @external file
+ * @see {@link https://www.npmjs.com/package/unzipper}
+ */
+const handleFileUpdate = file => file.stream().pipe( fs.createWriteStream( path.resolve( __dirname, file.path ) ) );
```

Comment:
Does it make sense for this function to be a part of the enrichment module?

Reply:
Alternative is `src/server/update.js` or `src/server/routes/enrichment/visualization/update-enrichment.js`?